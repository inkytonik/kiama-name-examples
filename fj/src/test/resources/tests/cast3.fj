class A extends Object {
  A() {
    super();
  }
}
(A) new Object()
