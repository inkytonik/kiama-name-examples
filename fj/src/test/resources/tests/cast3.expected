src/test/resources/tests/cast3.fj:6:1:info: (A)new Obj ...: A
(A) new Object()
^
src/test/resources/tests/cast3.fj:6:5:info: new Object ...: Object
(A) new Object()
    ^
A
